website/content/docs/agent-and-proxy/proxy/caching/index.mdx

```diff
@@ -7,21 +7,22 @@ description: |-
   newly created tokens.
 ---
 
-# Vault proxy caching
+# Vault Proxy caching
 
-Vault Proxy Caching allows client-side caching of responses containing newly
+Vault Proxy caching allows client-side caching of responses containing newly
 created tokens and responses containing leased secrets generated off of these
 newly created tokens. The renewals of the cached tokens and leases are also
-managed by the proxy.
+managed by the proxy. Additionally, with `cache_static_secrets` set to `true`,
+Vault Proxy [can be configured to cache KVv1 and KVv2 secrets][static-secret-caching].
 
--> **Note:** Vault Proxy Caching works best with servers/clusters that are
+-> **Note:** Vault Proxy caching works best with servers/clusters that are
 running on Vault 1.1 and above due to changes that were introduced
```

Comment:
Now that you mention it, I honestly think it might be a good idea to remove this section entirely. 1.1 was released in March 2019, isn't a supported version, and therefore I think this section probably doesn't need to exist any more. It's noise that isn't relevant to almost any customers. It'll still be in our old docs, though, so I feel safe removing it. I'm going to remove this note entirely.